galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -170,7 +169,9 @@ def _check_path_matches_expected_repo(path, filename_ns):
         if INBOUND_REPO_NAME_FORMAT.format(namespace_name=filename_ns) == repo_name:
             return
         raise NotFound(
-            f'Path does not match: "{INBOUND_REPO_NAME_FORMAT.format(namespace_name=filename_ns)}"')
+            _('Path does not match: '
+              f'"{INBOUND_REPO_NAME_FORMAT.format(namespace_name=filename_ns)}"')
```

Comment:
this code has a mix of f-strings and .format(), and it looks like f-strings are not supported with gettext